Web/HTML1/MetaGurukul.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Insert title here</title>
+</head>
+<body>
+	<form>
+	<fieldset form="">
```

Comment:
Why do you have to specify form="" when you are using this field set only of this form.